src/pages/Main/Main.tsx

```diff
@@ -71,7 +72,7 @@ const Main = () => {
         isComparisonMode={isComparisonModeOn}
         title={selected?.properties.NM_MUN}
       />
-
+      <SideInformationsContainer/>
```

Comment:
Please add a space between the components

Reply:
Spaces added.